src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -3806,6 +3823,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 ' symbol. This would reduce the error count, but may lead to a new diagnostics once the 
                 ' previous one was fixed (byref + return type).
 
+                If (comparisonResults And SymbolComparisonResults.TupleNamesMismatch) <> 0 Then
+                    diagnostics.Add(ErrorFactory.ErrorInfo(ERRID.ERR_DuplicateProcDefWithDifferentTupleNames, firstMember), location)
```

Comment:
>ERR_DuplicateProcDefWithDifferentTupleNames [](start = 65, length = 43)

All other similar errors in this function mention both members, consider doing the same.